Sources/SIL/SILPrinter.swift

```diff
@@ -537,6 +537,8 @@ class SILPrinter: Printer {
         case let .attributedType(attrs, type):
             print("", attrs, " ", " ") { print($0) }
             naked(type)
+        case .coroutineTokenType:
+            print("!CoroutineTokenType!")
```

Comment:
This won't parse, right?

Reply:
It won't, but it also doesn't have to. There's no way to express this type in the textual SIL format.